solutions/remotemonitoring/single-vm/docker-compose.java.yml

```diff
@@ -151,6 +153,22 @@ services:
       - PCS_AUTH_AUDIENCE
       - PCS_APPLICATION_SECRET
 
+  diagnostics:
+    image: azureiotpcs/pcs-diagnostics-dotnet:millennium
```

Comment:
update "millennuum" tag

Reply:
I actually want to use millennium tag as this shouldn't change for RMv2. Lets discuss.